icu4c/source/i18n/unicode/uspoof.h

```diff
@@ -1159,6 +1179,56 @@ uspoof_areConfusable(const USpoofChecker *sc,
                      const UChar *id2, int32_t length2,
                      UErrorCode *status);
 
+/**
```

Comment:
you need to wrap these new functions with

```
#ifndef U_HIDE_DRAFT_API
```
and
```
#endif /* U_HIDE_DRAFT_API */
```
See https://github.com/unicode-org/icu/commit/27181e36a6bed372482e5a11c25009fa4dd971f1#diff-b9219dc4c2592180776f39139e72abfd4a9dc27cb0a7adb128b9c76b7bd7d65eR16 as example